doc/mr.txt

```diff
@@ -175,6 +175,14 @@ mr#mrw#stop_recording()
 mr#mrr#stop_recording()
 	Stop MRU/MRW/MRR recording by several |autocmd|.
 
+						*mr#mru#delete()*
+						*mr#mrw#delete()*
+						*mr#mrr#delete()*
+mr#mru#delete(filename)
+mr#mrw#delete(filename)
+mr#mrr#delete(filename)
+	Delete a filename from the MRU/MRW/MRR list.
```

Comment:
Please do not hide specs if the function support both string and list. Additionally, please use `{}` for argumet like

```
mr#mru#delete({filename})
      Delete a {filename} from the ...
```

Reply:
fixed it: 2ef26fa